doc/Type/Int.pod6

```diff
@@ -36,8 +36,9 @@ respectively:
 
 These notations allow you to use variables, too:
 
-    :16($ninety-nine) # 153
-    :100[99, $two, 3] # 990203
+    my Int $ininety-nine = 99;
+    :16($ninety-nine); # 153
+    :100[99, $two, 3]; # 990203
```

Comment:
Looks like there's a small typo introduced in the first line addition here (`$ininety-nine`).